src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MinMaxTable.cs

```diff
@@ -23,5 +28,20 @@ public MinMaxTableFunction(bool isMin)
         {
             _delegationCapability = isMin ? DelegationCapability.Min : DelegationCapability.Max;
         }
+
+        public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
```

Comment:
binding is unused, consider overriding the CheckInvocation that doesn't require binding

Reply:
I tried removing it, but it caused the app to crash. All the other functions seem to do it this way.